src/services/rpcDnsZones.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  api,
+  Command,
+  getBatchCommand,
+  BatchRPCResponse,
+  FindRPCResponse,
+  getCommand,
+} from "./rpc";
+// utils
+import { API_VERSION_BACKUP } from "../utils/utils";
+// Data types
+import { DNSZone, dnsZoneType } from "src/utils/datatypes/globalDataTypes";
+import { apiToDnsZone } from "src/utils/dnsZonesUtils";
+
+/**
+ * DNS zones-related endpoints: useDnsZonesFindQuery, useGetDnsZonesFullDataQuery,
+                            useSearchDnsZonesEntriesMutation,
+ *
+ * API commands:
+ * - dnszone_find: https://freeipa.readthedocs.io/en/latest/api/dnszone_find.html
+ * - dnszone_show: https://freeipa.readthedocs.io/en/latest/api/dnszone_show.html
+ */
+
+export interface DnsZonesFindPayload {
+  searchValue: string;
+  pkeyOnly: boolean;
+  sizeLimit: number;
+  version?: string;
+}
+
+export interface DnsZonesFullDataPayload {
+  searchValue: string;
+  apiVersion: string;
+  sizelimit: number;
+  startIdx: number;
+  stopIdx: number;
+}
+
+export interface DNSZoneBatchResponse {
```

Comment:
Consider Pascal-case
```suggestion
export interface DnsZoneBatchResponse {
```